features/favorite.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Favorite
+
+  Scenario: Favorite Creation
+    Given I am logged into the site
+    When I visit a user's profile
+    And I click "Follow"
+    When I visit the "/users/following" page
```

Comment:
Should this be an `And`? This is a style thing, so @dkniffin  or someone else that has written more integration here might have more to say about SB style on this, but generally I think it's a good idea to have one `Given`, `When`, and `Then` in a scenario and use `And` for the rest..